lldb/test/API/lang/swift/expression/self_from_register/TestSelfFromRegister.py

```diff
@@ -0,0 +1,17 @@
+# TestSelfFromRegister.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
```

Comment:
Please remove the entire header. We no longer require this for tests and it's also much less useful since LLVM also moved to the Apache license.